tests/providers/node/base_model_test.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from ethtx.providers.node.base import NodeBase
+
+NODE = "test"
+POA = False
+
+
+def test_node_base_model():
+    node_base_data = NodeBase(NODE, POA)
+
+    assert node_base_data.node == NODE
+    assert node_base_data.poa == POA
+
+    assert repr(node_base_data) == f"<Node: {NODE}, poa: {POA}>"
+    assert str(node_base_data) == f"<Node: {NODE}, poa: {POA}>"
+
+    assert dict(node_base_data) == {"node": NODE, "poa": POA}
```

Comment:
Unit tests should hold a singular responsibility, here are tested thing like construction of the object, representation, iteration. Every single thing here should have it's own test. Also those tests should be in one class